tests/in/pointer-access.param.ron

```diff
@@ -0,0 +1,5 @@
+(
```

Comment:
any particular reason we are adding a SPIR-V test here (on the input)?
I'd think that bounds checking code is 99% in the backends (as the way we are trying to get it here), so the format of the input test shouldn't matter, in which case we should strongly prefer WGSL on the input.

Can this logic be just merged into the existing `access.wgsl`?

Reply:
This is SPIR-V only because our WGSL front end doesn't implement the `*` dereferencing operator yet. Once we implement that, it can be replaced with WGSL.